local-cli/generator-android/templates/src/app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,7 @@
     package="<%= package %>">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
```

Comment:
I wonder if Gradle adds this permission automatically.
Without it the buck compiled APK throws.